releasenotes/notes/profiling-send-classnames-3a65dc5b33be5b0a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    profiling: adds classnames to each frames, when possible
```

Comment:
```suggestion
    profiling: adds the class name to each frame, when possible
```

Reply:
@emmettbutler went ahead and took this, thanks!